src/containers/NibrsContainer.js

```diff
@@ -117,8 +117,7 @@ const NibrsContainer = ({
       {content}
       {isReady &&
         <div className="serif italic fs-12">
-          Source: Reported {nibrsTerm} data from {placeDisplay},{' '}
-          {nibrsFirstYear}–{until}.
+          Source: Reported {nibrsTerm} data from {placeDisplay}.
```

Comment:
do you think it's important to have that in the source? I excluded so i didn't have to add that special `nibrsFirstYear` logic across files